.github/workflows/macosx_windows_ci.yaml

```diff
@@ -6,8 +6,10 @@ jobs:
     name: Run Tests
     env:
       PYTHON: ${{ matrix.python-version }}
-      OS: ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
+    defaults:
+     run:
+       shell: bash -l {0}
```

Comment:
what does this do?

Reply:
so the `-l` adds the `login` flag to the bash shell. I've attached the clip from the man page about what that option does. The real reason I added it is because conda doesn't always seem to work correctly on windows without it. Nothing I have noticed on this particular repository but I have on 21cmsense. Inspecting the examples from the miniconda setup action's github page they also used the `-l` option in their examples.

>When bash is invoked as an interactive login shell, or as  a  non-interactive  shell  with  the
       --login  option,  it first reads and executes commands from the file /etc/profile, if that file
       exists.  After reading that file, it looks for ~/.bash_profile, ~/.bash_login, and  ~/.profile,
       in  that order, and reads and executes commands from the first one that exists and is readable.
       The --noprofile option may be used when the shell is started to inhibit this behavior.

Reply:
maybe add a comment that it's required to make windows work properly?